doc/source/about.rst

```diff
@@ -0,0 +1,110 @@
+
+An example IMPROVER forecast product is shown below, here for wind speed. Four fields are plotted at different wind speed thresholds that may be of interest to the user, each field shows the probability of exceeding the relevant threshold.
+
+
+.. figure:: ../files/wind_probabilities.jpg
+   :align: center
+
+   Probability of exceeding wind speed thresholds
+
+
+Structure of IMPROVER
+---------------------
+
+IMPROVER is designed as a modular post-processing system. The final product is created through the application of a sequence of processing steps, where the sequence can be readily modified to achieve different outcomes. The output of each step in the chain can be written out, allowing for verification against observations at each stage (using a suitable verification package, which is not part of IMPROVER). This enables the user to determine whether a given step in the chain is improving or harming the forecast quality.
+
+Any given step in the processing can be applied using the included command line interfaces (CLIs). A complex system is built by calling the CLIs in sequence. The simple schematic below gives an example processing chain.
+
+.. figure:: ../files/processing_chain.jpg
+   :align: center
+
+   An example processing chain with IMPROVER
+
+
+Using IMPROVER
+--------------
+
+IMPROVER does not currently include installation functionality (e.g. setup.py). The code can be used by cloning the `GitHub repository`_ and calling the command line interfaces (CLIs) from a linux/unix terminal or by importing the modules directly into Python.
+
+.. _GitHub repository: https://github.com/metoppv/improver
+
+.. code:: console
+
+    git clone https://github.com/metoppv/improver.git <local directory>
+
+The list of dependencies can be found in the `environment.yml`_ file.
+
+.. _environment.yml: https://github.com/metoppv/improver/blob/master/environment.yml
+
+Example use of a CLI
+====================
+
+Here we give a simple example of using an IMPROVER CLI to threshold data, moving into probability space. This CLI invocation is from the root directory of a local copy of IMPROVER.
+
+.. code:: console
+
+    bin/improver threshold input_file.nc output_file.nc 5 10 15 20 --threshold_units m/s
+
+
+* input_file.nc is a netCDF file containing a forecast diagnostic data cube, e.g. wind speeds across an x-y grid at a given time.
+* for each threshold specified (5, 10, 15, 20 m/s) a new x-y grid of data will be created. Each point in the grid will contain a 0 if the input wind speed at that point was below the threshold, or 1 if it was above the threshold.
+* output_file.nc will be a new netCDF file containing the resulting data cube with an additional leading dimension that corresponds to the given thresholds (5, 10, 15, 20 m/s).
+
+This simple example covers one step in a processing chain. Additional information about using any CLI can be found on the command line using `-h`, e.g.:
+
+.. code:: console
+
+    bin/improver nbhood -h
+
```

Comment:
Add: "The current list of CLIs can be found \<a href='improver.cli.html'>here</a>"

Reply:
Done.